### PR TITLE
feat: add brand colabration section

### PR DESCRIPTION
**Pull request by rpandox**
**This Pull Request resolves issue #38**

### Adds

- brands colabration section
- makes it responsive

https://github.com/user-attachments/assets/9672c92a-edb8-45ac-ab5e-ce0c58891711

### CODE REVIEW

**Review by prabhavpokharel — CHANGES_REQUESTED** [review]
@rpandox , can you make the section heading match with other section headings in the homepage? I recommend using Camel Casing in the heading. Like this,
```html
<h2 class="text-5xl font-bold text-[{{site.text-colors.darkblue}}] text-center mb-8"> Upcoming Events </h2>
```
Also, try to keep a light gray color as bg to make it look distinct from the homepage bg.
```
gray: "#E0E5F5"
```

**Comment by prabhavpokharel** [comment]
And a thought regarding the layout: adding a bit more padding at the bottom could enhance the overall appearance. While it’s not mandatory, I believe it might improve the visual balance.

**Comment by rpandox** [comment]
will work on it

**Comment by rpandox** [comment]
@prabhavpokharel , the changes requested were done
please review it

**Review by prabhavpokharel — APPROVED** [review]
Nice job!

File: _includes/brands_colab.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<section id="Brands-Collaboration">
+  <div
+    class=" bg-[{{site.bg-colors.gray}}] flex flex-col justify-center gap-3 py-12 mt-4"
+  >
+    <div
+      class="text-5xl font-bold text-[{{site.text-colors.darkblue}}] text-center mb-8"
+    >
+      Brands we have collaborated with
```
Comment (aayush105) [inline _includes/brands_colab.html:8]: write Brand We Have Collaborated With (case sensitive) in the heading.

**Review by aayush105 — CHANGES_REQUESTED** [review]

**Review by aayush105 — APPROVED** [review]

**Review by gaurovgiri — APPROVED** [review]
Awesome work

**Pull request merged**